package.json

```diff
@@ -55,8 +55,9 @@
     "autoprefixer": "^10.0.1",
     "babel-plugin-react-css-modules": "^5.2.6",
     "bootstrap": "^4.5.3",
-    "precss": "^4.0.0",
+    "bootstrap-css-modules": "^4.2.1",
     "formik": "^2.2.1",
+    "precss": "^4.0.0",
```

Comment:
?

Reply:
packege.json updated according to your comment